misc/snapshotter/snapshotter.sh

```diff
@@ -97,9 +104,53 @@ function options_handler() {
 
 }
 
+function configure_snapshotter() {
+
+    if [ "${CONTAINER_RUNTIME}" != "containerd" ]; then
+        die "not supported container runtime: ${CONTAINER_RUNTIME}"
+    fi
+
+    # Copy the container runtime config to a backup
+    cp "$CONTAINER_RUNTIME_CONFIG" "$CONTAINER_RUNTIME_CONFIG".bak.nydus
+
+    # Check and add nydus proxy plugin in the config
+    if toml check $CONTAINER_RUNTIME_CONFIG "proxy_plugins" >/dev/null; then
+        if [ "$(toml get $CONTAINER_RUNTIME_CONFIG proxy_plugins | jq length)" == "0" ]; then
+            echo "there is no proxy plugin!"
+            cat <<EOF >>"$CONTAINER_RUNTIME_CONFIG"
+    [proxy_plugins.nydus]
+        type = "snapshot"
+        address = "/run/containerd-nydus/containerd-nydus-grpc.sock"
+EOF
+        elif toml check $CONTAINER_RUNTIME_CONFIG "proxy_plugins.nydus" >/dev/null; then
+            echo "the config has configured the nydus proxy plugin!"
+        else
+            die "there is another proxy plugin in the containerd!"
```

Comment:
Can't it be multiple proxy plugins?

Reply:
Thanks for your reminder, I have fixed it.